examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+  <link href="https://fonts.googleapis.com/css?family=Questrial" rel="stylesheet" type="text/css">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+
+  <style amp-custom>
+    body {
+      margin: auto;
+      font-family: sans-serif;
+    }
+    h1.header {
+      background-color: red;
+      color: white;
+      text-align: center;
+      margin: 0;
+      padding: 10px;
+      font-size: 24px;
+    }
+    .container {
+      padding: 0 10px;
+    }
+    button.buy {
+      margin: 20px 0;
+      width: 100%;
+      padding: 10px;
+      background-color: green;
+      color: white;
+      font-weight: bold;
+    }
+    button[disabled] {
+      background-color: red;
+    }
+    .thumbnails, .color {
+      margin-top: 10px;
+    }
+    .thumbnails amp-img {
+      border: solid 2px transparent;
+      padding: 2px;
+    }
+    .thumbnails amp-img[selected] {
+      border: solid 2px gray;
+    }
+    .thumbnails amp-img[out-of-stock] {
+      background-color: red;
+    }
+    amp-accordion {
+      margin-top: 10px;
+    }
+    .sizes button {
+      display: block;
+      width: 100%;
+      border: solid 1px gray;
+      padding: 5px;
+      background-color: #eee;
+    }
+    .change-size {
+      display: block;
+      width: 200px;
+      height: 30px;
+      padding: 5px;
+      background-color: #666;
+      color: white;
+    }
+    button[hidden] {
+      display: none;
+    }
+    p[hidden] {
+      display:none;
+    }
+  </style>
+</head>
+
+<body>
+  <h1 class="header">AMPWEAR</h1>
+  <div class="container">
+    <h2>MEN'S COTTON SHORT SLEEVE T-SHIRT</h2>
+
+    <amp-bind-dataset layout=nodisplay>
```

Comment:
I'd really like to have an `id` here. It might seem a small thing, but I think it's a very important aspect and the one I failed to fully communicated before.